winui/Shimmer/Overview.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: Overview of WinUI SfShimmer control | Syncfusion
```

Comment:
@GilbertmanojSF3971 title: Overview of WinUI Shimmer control | Syncfusion

Reply:
@Ganesh-Mariappan Changed the SfShimmer into Shimmer in the tiltle.